datafusion/src/logical_plan/builder.rs

```diff
@@ -137,20 +138,20 @@ impl LogicalPlanBuilder {
     pub fn scan_parquet(
         path: impl Into<String>,
         projection: Option<Vec<usize>>,
-        max_concurrency: usize,
+        context: ExecutionContext,
```

Comment:
I don't have a better solution on how to handle this, but it strikes me a bit odd to couple execution context with logical plan builder here. I will think more about this later this week.

Reply:
Perhaps we could  pass in the ObjectStore / ObjectStoreRegistry reference rather than an ExecutionContext